All-in-one setup/createform.ps1

```diff
@@ -627,20 +628,32 @@ try {
 
         $exchangeSession = Connect-ExchangeOnline -Credential $credential -ShowBanner:$false -ShowProgress:$false -TrackPerformance:$false -ErrorAction Stop 
 
-        Hid-Write-Status -Event Information -Message "Successfully connected to Office 365"
+        Write-Information -Message "Successfully connected to Office 365"
+
+        $Log = @{
+            Action            = "MailboxPermissions" # optional. ENUM (undefined = default) 
+            System            = "ExchangeOnline" # optional (free format text) 
+            Message           = "Successfully connected to Office 365" # required (free format text) 
```

Comment:
Only log audit messages when you're doing a write action to a system.

Reply:
Changed